### PR TITLE
update docs to make mkdocs page

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
I've added three new files  in 'docs':
index.md
Contribute-to-documentation.md
Developer-documentation-style-guide.md
changed some layout in get-started.md
After the doc is done, I will make a doc site based on that.

### CODE REVIEW

File: docs/Contribute-to-documentation.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+---
+## Contribute to documentation
+Contributing to the Nebula documentation can be a rewarding experience. When you offer feedback, questions, edits, or new content, you help us, the projects you work on, and the larger Nebula community.
+
+We welcome your participation to help make the documentation better!
+
+
+## How to contribute to the docs
+There are many wats to contribute:
+ - Edit, rate, or file an issue or question directly on the site by using the links available on the right-side menu on every pare at docs.nebula.com.
```
Comment (darionyaphet) [inline docs/Contribute-to-documentation.md]: docs.nebula.com ？

File: docs/get-started.md [hunk 2]
```diff
@@ -1,4 +1,3 @@
-## Get Started
```
Comment (darionyaphet) [inline docs/get-started.md]: I find the other `.md` file's name is upper case . Please rename `get-started.md` to `Get-started.md` thanks

File: docs/get-started.md [hunk 3]
```diff
@@ -22,7 +21,11 @@ docker --version
 
 If `docker` is not found, please see [here](https://docs.docker.com/install/) for more information to install docker.
 
-After that, using `docker pull vesoft/nebula-graph:latest` to get `nebula` image and `docker images` that can display images status.
+After that, using
+```
```
Comment (darionyaphet) [inline docs/get-started.md]: Using `block` is better.  :) I like it .

File: docs/Developer-documentation-style-guide.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
+---
```
Comment (darionyaphet) [inline docs/Developer-documentation-style-guide.md]: Is this the header?  I think it's a good habit. 

Would you mind adding it to other documents? thanks

File: docs/Contribute-to-documentation.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+
+---
+## Contribute to documentation
+Contributing to the Nebula documentation can be a rewarding experience. When you offer feedback, questions, edits, or new content, you help us, the projects you work on, and the larger Nebula community.
+
+We welcome your participation to help make the documentation better!
+
+
+## How to contribute to the docs
+There are many wats to contribute:
```
Comment (dutor) [inline docs/Contribute-to-documentation.md]: what's _wats_?

File: docs/Contribute-to-documentation.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+
+---
+## Contribute to documentation
+Contributing to the Nebula documentation can be a rewarding experience. When you offer feedback, questions, edits, or new content, you help us, the projects you work on, and the larger Nebula community.
```
Comment (dutor) [inline docs/Contribute-to-documentation.md]:
```
When you offer feedback, questions, edits, or new content, you help us, the projects you work on, and the larger Nebula community.
```
This should be divided into two.

_When you offer feedbacks, questions, edits, or new contents, you are helping us._

Besides, what _the projects you work on, and the larger Nebula community._  is saying?

File: docs/Contribute-to-documentation.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+
+---
+## Contribute to documentation
```
Comment (dutor) [inline docs/Contribute-to-documentation.md]: The filename of docs should be all lower case.

File: docs/get-started.md (hunk 2 above)
Comment (dutor) [reply docs/get-started.md]: If the filename will be showed in the address bar of the browser, lower case might be more suitable, except some special ones, like `nGQL`.

File: docs/how-to-contribute.md [hunk 8]
```diff
@@ -53,7 +53,7 @@ This hook checks your commits for formatting, building, doc generation, etc.
 
 ```sh
 cd $working_dir/nebula/.git/hooks
-ln -s ../../.linters/cpp/hooks/pre-commit.sh pre-commit
+ln -s ../../cpplint/bin/pre-commit.sh .
```
Comment (dutor) [inline docs/how-to-contribute.md]: Why this change?

File: docs/Contribute-to-documentation.md (hunk 6 above)
Comment (Amber1990Zhang) [reply docs/Contribute-to-documentation.md]: Means users' feedback can help us improve our Nebula community. Deleted that sentence and add
We welcome your participation to help make the documentation better!

File: docs/get-started.md (hunk 2 above)
Comment (Amber1990Zhang) [reply docs/get-started.md]: fixed

File: docs/Contribute-to-documentation.md (hunk 5 above)
Comment (Amber1990Zhang) [reply docs/Contribute-to-documentation.md]: it's a typo, it should ways, thank you

File: docs/Contribute-to-documentation.md (hunk 7 above)
Comment (Amber1990Zhang) [reply docs/Contribute-to-documentation.md]: fixed

**Comment by dutor** [comment]
Jenkins, go!
